lib/evaluation/bundlers/inputDescriptorsEB.ts

```diff
@@ -0,0 +1,98 @@
+import { InputDescriptor } from '@sphereon/pe-models/model/inputDescriptor';
+import Ajv from 'ajv';
+import jp from 'jsonpath';
+
+import { Evaluation } from '../core';
+
+import { EvaluationBundler } from './evaluationBundler';
+
+export class InputDescriptorsEB extends EvaluationBundler<unknown, InputDescriptor[]> {
+  constructor(parentTag: string) {
+    super(parentTag, 'input_descriptors');
+  }
+
+  public getEvaluations(d: any, p: InputDescriptor[]): Evaluation<any, any>[] {
+    return [...this.inputCandidateEvaluations(d, p)];
+  }
+
+  private inputCandidateEvaluations(d: any, p: InputDescriptor[]): Evaluation<any, any>[] {
+    return [
+      {
+        tag: this.getTag(),
+        target: { d, p },
+        predicate: () => d != null && p != null,
+        message: 'input_definition should not be null',
+      },
+      {
+        tag: this.getTag(),
+        target: { d, p },
+        predicate: InputDescriptorsEB.evaluateCandidates,
+        message: 'input candidate is invalid',
+      },
+    ];
+  }
```

Comment:
we do not need to define a separate method for this. Can be edited as follows:

public getEvaluations(d: any, p: InputDescriptor[]): Evaluation<any, any>[] {
    return [
      {
        tag: this.getTag(),
        target: { d, p },
        predicate: () => d != null && p != null,
        message: 'input_definition should not be null',
      },
      {
        tag: this.getTag(),
        target: { d, p },
        predicate: InputDescriptorsEB.evaluateCandidates,
        message: 'input candidate is invalid',
      },
    ];
  }